manager/manager.go

```diff
@@ -101,6 +101,10 @@ const (
 
 func (s *SyncManager) Start() error {
 
+	if logUtils.IsCleanOnStartup() {
```

Comment:
you should rename this to `ShouldCleanOnStartup`

Reply:
Sounds good